developmentseed/pearl-frontend
# Load latest timeframe on AOI or mosaic change

Fix error described at https://github.com/developmentseed/pearl-frontend/pull/57#issuecomment-1598618475.

How to test:

- Load existing project
- Change mosaic, it should load the latest timeframe, if exists
- Change AOI and mosaic, it should load latest timeframe

@geohacker ready for review.
